coresdk/src/coresdk/input.cpp

```diff
@@ -56,8 +56,9 @@ namespace splashkit_lib
         if( ! _input_callbacks.do_quit )
         {
             _input_callbacks.do_quit              = &quit;
-            _input_callbacks.handle_key_down      = &_handle_key_down_callback;
-            _input_callbacks.handle_key_up        = &_handle_key_up_callback;
+            // TODO: Temporarily do not call due to 70c30d4
```

Comment:
These callbacks are entirely within the coresdk -- so they shouldn't have changed.